src/core/public/chrome/ui/header/collapsible_nav_group_enabled_top.tsx

```diff
@@ -125,12 +152,10 @@ export const CollapsibleNavTop = ({
         <>
           <EuiSpacer />
           <EuiText>
-            <div className="nav-link-item" style={{ fontWeight: 'normal' }}>
-              {currentNavGroup?.title}
-            </div>
+            <div style={{ fontWeight: 'normal' }}>{currentNavGroup?.title}</div>
```

Comment:
Q: what's this title? why customize the font-weight?

Reply:
Nice catch, updated and thanks for that.